components/ShakeApi.js

```diff
@@ -0,0 +1,105 @@
+import React, {useState, useEffect} from "react";
+import { View, Text, TouchableOpacity, Image, StyleSheet } from "react-native";
+import { Accelerometer } from "expo-sensors";
+
+const styles = StyleSheet.create({
+    container: {
+        padding: 25,
+        alignItems: "center",
+        justifyContent: "center",
+        },
+        tinyFrame: {
+        width: 300,
+        height: 300,
+        marginBottom: 25,
+        },
+        button: {
+            width: 125,
+            backgroundColor: "black",  
+            alignItems: "center",
+            padding: 10,
+            borderRadius: 25,
+        },
+        text: {
+            color: "white",
+        }
+});
+
+const url = `https://randomfox.ca/floof/`
```

Comment:
Love the choice of api, clever to use something so cute which already does half the job with getting the app to look good ;)